src/System.Text.Encoding/tests/Performance/System.Text.Encoding.Performance.Tests.csproj

```diff
@@ -6,9 +6,9 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <ProjectGuid>{859C92CB-72FB-453C-A363-7CD025E29B1D}</ProjectGuid>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcoreapp-Debug|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcoreapp-Release|AnyCPU'" />
   <!-- Default configurations to help VS understand the configurations -->
```

Comment:
I don't care, but it moved above this comment. I guess we could remove all such comments

Reply:
yeah I've been just deleting those comments overtime

Reply:
Ok. I just wrote a small tool that deletes all the comments from remaining csprojs having those comments. Will update the PR so that we don't have those comments anymore.